app/jobs/batch_update_job.rb

```diff
@@ -77,6 +77,36 @@ def update_file(gf, user)
     end
     save_tries = 0
     begin
+      ark_identifier = Ezid::Identifier.create(id: Ezid::Client.config.default_shoulder + gf.id)
```

Comment:
Shouldn't we use gf.ark_id here, instead of recreating the ARK from the shoulder and the NOID? We always want the update to go the ARK that was previously registered, and if we ever use different shoulders, this will break.